app/controllers/mock.php

```diff
@@ -646,6 +649,69 @@
         $response->noContent();
     });
 
+App::get('/v1/mock/github/callback')
+    ->desc('Create installation document using GitHub installation id')
+    ->groups(['mock', 'api', 'vcs'])
```

Comment:
Let's only keep `mock` group here, unless we really need other ones for something